src/main/haskell/kore/src/Kore/Builtin/List.hs

```diff
@@ -188,7 +188,7 @@ evalGet =
                     if Seq.null _list
                         then Builtin.appliedFunction ExpandedPattern.bottom
                         else empty
-                bothConcrete = do
+                bothConcreteVariable = do
```

Comment:
As my comment above, I think including `Variable` in the name here is very misleading.